Cogs/ServerManagement.py

```diff
@@ -255,13 +282,16 @@ async def rolemenu(self, ctx):
 
     @commands.Cog.listener()
     async def on_button_click(self, res):
+        """Handles button clicks for role menus
+        If button clicked is a role menu message,
+        Get the role name from the button label
+        Get role from role name
+        If role doesn't exist, error
+        If role does exist, apply or remove the role to the member who clicked the button
+        """
```

Comment:
It might be worth documenting `res` since it's a slightly different setup than the normal `ctx` argument that the functions use